media/css/firefox/family/components/modules/_privacy.scss

```diff
@@ -19,4 +19,39 @@
             background-color: f3.$green-light;
         }
     }
+
+    @media #{$mq-sm} {
+        .l-grid {
+            display: grid;
+
+            .c-subtitle {
+                min-width: 8ch;
+                max-width: 12ch;
+            }
+        }
+    }
+
+    @media #{$mq-lg} {
+        .l-grid {
+            grid-template-columns: minmax(20ch, 35ch) 1fr;
+            grid-template-rows: 100px auto auto;
+            column-gap: $layout-md;
+        }
+
+        .c-content {
+            grid-row: 2;
+        }
+
+        .c-browser {
+            grid-row: 1 / 3;
+            height: min-content;
+            margin: 0 auto;
+            max-width: 525px;
```

Comment:
**issue**: the browser grows fairly large on medium screens

**suggestion**: set max-width for browser on main styles, around L17, because we'll always want to keep it below a certain width